### PR TITLE
Homepage redesign & refactor, including marketing text content block …

### PR DESCRIPTION
**Pull request by seanaery**
…& images. Closes RDR-365, RDR-366, RDR-367, RDR-360.

### CODE REVIEW

File: app/views/hyrax/homepage/_home_marketing_boxes.html.erb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<div class="row home-marketing-boxes">
+  <div class="col-md-3 col-xs-6 marketing-box">
+    <%= image_tag("icons/magazine.png", class: 'img-responsive', alt: 'magazine icon') %>
+    <div class="marketing-caption">
+      Fulfill funder and journal requirements
```
Comment (mikedaul) [inline app/views/hyrax/homepage/_home_marketing_boxes.html.erb]: Should this kind of content be housed in the translations file?

Comment (seanaery) [reply app/views/hyrax/homepage/_home_marketing_boxes.html.erb]: @mikedaul Great point! I'll do that and resubmit...

**Comment by seanaery** [comment]
Here's a screenshot, with some Marketing Text content added via the corresponding content block in the Dashboard.
rdr-home-full

**Comment by seanaery** [comment]
Mobile style screenshots:

rdr-home-xs

rdr-home-sm

**Pull request merged**